graphistry/ArrowFileUploader.py

```diff
@@ -109,72 +117,114 @@ def post_arrow(self, arr: pa.Table, file_id: str, url_opts: str = 'erase=true')
 
         try:
             out = res.json()
-            logger.debug('Server upload file response: %s', out)
-            if not out['is_valid']:
-                if out['is_uploaded']:
-                    raise RuntimeError("Uploaded file contents but cannot parse (file_id still valid), see errors", out['errors'])
+            logger.debug("Server upload file response: %s", out)
+            if not out["is_valid"]:
+                if out["is_uploaded"]:
+                    raise RuntimeError(
+                        "Uploaded file contents but cannot parse "
+                        "(file_id still valid), see errors",
+                        out["errors"],
+                    )
                 else:
-                    raise RuntimeError("Erased uploaded file contents upon failure (file_id still valid), see errors", out['errors'])
+                    raise RuntimeError(
+                        "Erased uploaded file contents upon failure "
+                        "(file_id still valid), see errors",
+                        out["errors"],
+                    )
             return out
         except Exception as e:
-            logger.error('Failed uploading file: %s', res.text, exc_info=True)
+            logger.error("Failed uploading file: %s", res.text, exc_info=True)
```

Comment:
fwiw lint changes can likely be reverted... maybe we should have a flake setting to help standardize & avoid this kind of review noise?